### ISSUE TITLE
validate_poi.py different result with python3.7

### ISSUE THREAD

**Issue by pdibarto** [udacity/ud120-projects#220]
i'm solving mini-project on Validation and i'm using Python 3.7.
In this case i had different accuracy result in my validate.py with respect to Udacity course.
my code is the following:

import pickle
import sys
sys.path.append("../tools/")
from feature_format import featureFormat, targetFeatureSplit
from sklearn import tree
from sklearn.metrics import accuracy_score
from sklearn.model_selection import train_test_split

data_dict = pickle.load(open("../final_project/final_project_dataset_unix.pkl", "rb"))
features_list = ["poi", "salary"]
data = featureFormat(data_dict, features_list)
labels, features = targetFeatureSplit(data)

# it's all yours from here forward!
features_train, features_test, labels_train, labels_test = \
    train_test_split(features, labels, test_size=0.3, random_state=42)
clf = tree.DecisionTreeClassifier()
clf = clf.fit(features_train, labels_train)
clf_pred = clf.predict(features_test)
clf_acc = accuracy_score(clf_pred, labels_test)
print("accuracy score with train/test split:", clf_acc)

my result is: **accuracy score with train/test split: 0.6896551724137931**
and it is not correct according to Udacity course.
i do not understand what is wrong in my code. Could the issue be related to the function train_test_split which is changed in Python 3.7?

Thanks in advance for the help

**Comment by pdibarto** [comment]
i solved adding sort_keys in feature format :-)

**Comment by georgereimer8** [comment]
sort_keys = None ... no wait!
sort_keys = True  .. not wait!
sort_keys = '../tools/python2_lesson06_keys.pkl' .. no wait!
sort_keys = '../tools/python2_lesson14_keys.pkl' ... Sheesh! Python shure lets you write sloppy code. Its at times like this I appreciated strongly type objects.

**Comment by dario25494** [comment]
> i solved adding sort_keys in feature format :-)



> sort_keys = None ... no wait! sort_keys = True .. not wait! sort_keys = '../tools/python2_lesson06_keys.pkl' .. no wait! sort_keys = '../tools/python2_lesson14_keys.pkl' ... Sheesh! Python shure lets you write sloppy code. Its at times like this I appreciated strongly type objects.

Hi, where did you add this line in the code?
thanks in advance
